module/epochs/machine_account.go

```diff
@@ -37,28 +37,58 @@ const (
 
 func init() {
 	var err error
-	SoftMinBalanceLN, err = cadence.NewUFix64("0.0025")
+	defaultSoftMinBalanceLN, err = cadence.NewUFix64("0.0025")
 	if err != nil {
 		panic(fmt.Errorf("could not convert soft min balance for LN: %w", err))
 	}
-	HardMinBalanceLN, err = cadence.NewUFix64("0.002")
+	defaultHardMinBalanceLN, err = cadence.NewUFix64("0.002")
 	if err != nil {
 		panic(fmt.Errorf("could not convert hard min balance for LN: %w", err))
 	}
-	SoftMinBalanceSN, err = cadence.NewUFix64("0.125")
+	defaultSoftMinBalanceSN, err = cadence.NewUFix64("0.125")
 	if err != nil {
 		panic(fmt.Errorf("could not convert soft min balance for SN: %w", err))
 	}
-	HardMinBalanceSN, err = cadence.NewUFix64("0.05")
+	defaultHardMinBalanceSN, err = cadence.NewUFix64("0.05")
 	if err != nil {
 		panic(fmt.Errorf("could not convert hard min balance for SN: %w", err))
 	}
 }
 
+// MachineAccountValidatorConfig defines configuration options for MachineAccountConfigValidator.
+type MachineAccountValidatorConfig struct {
+	SoftMinBalanceLN cadence.UFix64
+	HardMinBalanceLN cadence.UFix64
+	SoftMinBalanceSN cadence.UFix64
+	HardMinBalanceSN cadence.UFix64
+}
+
+func DefaultMachineAccountValidatorConfig() MachineAccountValidatorConfig {
+	return MachineAccountValidatorConfig{
+		SoftMinBalanceLN: defaultSoftMinBalanceLN,
+		HardMinBalanceLN: defaultHardMinBalanceLN,
+		SoftMinBalanceSN: defaultSoftMinBalanceSN,
+		HardMinBalanceSN: defaultHardMinBalanceSN,
+	}
+}
+
+// WithoutBalanceChecks sets minimum balances to 0 to effectively disable minimum
+// balance checks. This is useful for test networks where transaction fees are
+// disabled.
+func WithoutBalanceChecks(conf *MachineAccountValidatorConfig) {
+	conf.SoftMinBalanceLN = 0
+	conf.HardMinBalanceLN = 0
+	conf.SoftMinBalanceSN = 0
+	conf.HardMinBalanceSN = 0
+}
```

Comment:
maybe expose this method only in tests? Since it's only for local network.

Reply:
Well, it's for any transient network, so local integration tests, Localnet, and Benchnet. We use the same binary for Localnet/Benchnet as real networks, so I think it needs to be exposed for all of them.